resources/php/multiple-return-statements.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * McCabeComplexity test resource file
+ * Complexity test resource file
```

Comment:
We could remove the commentse by moving the file into a "complexity" folder. By that, we would not need to touch exisitng files e.g. when a metric is renamed.

Reply:
The comment is not needed at all, I guess. Most files do not include such a comment. This one did and setting the renaming tool of the IDE to change the name also in comments and strings has therefore also renamed that one.
Currently, some tests files are also used for multiple test cases. I found that quite useful to avoid redundancy and in the end, all metrics are (and should be able to be) calculated for each file. 

Thinking of it, we may restructure the test cases to make metric-gardener run once on all test files and then let the test cases run assertions on the results. That should speedup the tests runs and dramatically reduce the probability of the race condition to occur 🤔

Reply:
But that is not really related to this PR and could be done regardless of the folder structure.